### DISCUSSION TITLE
Quarto and sjPlot

### DISCUSSION THREAD

**Discussion by israelarevalo** [quarto-dev/quarto-cli#5092] [Q&A] [answered]
I just updated to the newest release of Quarto and my html tables created through sjPlot are not displaying correctly on my html documents. All libraries are updated on both my windows and Mac machines and they are both yielding the same issue. The html file displays the text the table is supposed to contain, but no formatting as usually contained within an sjPlot. 

Any ideas from the community?

**Comment by cderv** [comment]
Thanks for the report. Can you share a qmd code to repoduce using a sjPlot table ? This would help look into that.

We did a lot of table adjustment and it is hard to get all the specificity all table R packages could have.

**Reply by israelarevalo** [reply]
Certainly - the sample qmd is below.


````markdown

---
title: "Sample"
author: "Sample Author"
format: 
  html: 
    toc: true
    toc-depth: 2
---

```{r}
library(sjPlot)
library(lme4)

model <- lmer(mpg ~ disp + hp + (disp | cyl), data = mtcars)
sjPlot::tab_model(model) 
```
````

Please let me know if there's anything else needed.

**Reply by cscheid** [reply]
This is enough, thanks!

The issue here is that Quarto 1.3 sends HTML tables to Pandoc for processing, and that table is not being recognized by Pandoc successfully as a table. This is a new 1.3 feature: https://quarto.org/docs/prerelease/1.3/tables.html

I think we need to not be so aggressive here when Pandoc itself fails to parse tables. I'll see if I can improve this.

**Reply by israelarevalo** [reply]
Thank you! I appreciate the support.

**Reply by cscheid** [reply, marked as answer]
Pull request incoming: https://github.com/quarto-dev/quarto-cli/pull/5095

We'll create a new release shortly. Thanks for the report! In the fixed version, I see this:

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/285675/229937728-b41ba37a-2648-44b4-91a3-a53a800d2ffa.png">

**Comment by ghost** [comment]
Moved to https://github.com/quarto-dev/quarto-cli/issues/5097

**Reply by cderv** [reply]
A bit of a mix up, it was moved to #5094 before that 😅  - Adding to keep track of all references
